### PR TITLE
02.02.01, 02.02.02, 02.02.05

### PR DESCRIPTION
**Pull request by ColeDewis**
This PR should complete the 3 above issues.

Adds the ability to add profile photos, have them be stored in cloud storage, as well as deleting profile photos.

A profile photo will be autogenerated by default, and permanently stored.

### CODE REVIEW

**Comment by ColeDewis** [comment]
@ServingCoder you should be able to review this now, it's mostly just the code we had on the weekend but with some improvements. It probably will crash when it tries to generate a profile photo since that isn't ready yet, but you can look at the code and I can help you get around that for testing today.

This is currently missing the proper autogeneration, @WhoShotCupidd we need to properly get your autogeneration in before this is merged.

**Comment by WhoShotCupidd** [comment]
@ColeDewis ok sounds good ill work off this for autogeneration

File: code/noRAM/app/src/main/java/com/example/noram/model/Attendee.java [hunk 1]
```diff
@@ -132,23 +141,6 @@ public void setPhoneNumber(String phoneNumber) {
...
-    /**
-     * A method to get the profile picture of the attendee
-     * @return the profile picture of the attendee
-     */
-    public String getProfilePicture() {
-        return profilePicture;
-    }
-
-    /**
-     * A method to set the profile picture of the attendee
```
Comment (ServingCoder) [inline code/noRAM/app/src/main/java/com/example/noram/model/Attendee.java:140]: no longer need this because the database handles it

File: code/noRAM/app/src/main/java/com/example/noram/model/Database.java
Comment (ServingCoder) [inline code/noRAM/app/src/main/java/com/example/noram/model/Database.java:1]: @WhoShotCupidd Christiaan, please look at lines 91 to 133 and confirm if your "TODO" methods are even needed in our code anymore. This issue is not really related to the adding photos functionalities in this PR, so this is a general comment.

**Review by ServingCoder — APPROVED** [review]
Cole and I went through reviewing this pull request and made minimal changes to fix small bugs to the photo displaying in the attendee fragment.  We noticed methods related to Christiaan and the database that may no longer be needed and need revewing on it's own. Because that is not related to this current issue on the pull request, that can be for another time.

File: code/noRAM/app/src/main/java/com/example/noram/MainActivity.java [hunk 2]
```diff
@@ -95,17 +98,24 @@ private void signInFirebase() {
...
                                 DocumentSnapshot document = task1.getResult();
+
                                 if (document.exists()) {
                                     // If the user exists, get the user's information
                                     String firstname = document.getString("firstName");
                                     String lastname = document.getString("lastName");
                                     String homepage = document.getString("homePage");
-                                    String email = document.getString("email");
-                                    String profilePicture = document.getString("profilePicture");
+                                    String phoneNumber = document.getString("phoneNumber");
```
Comment (ethanchung936) [inline code/noRAM/app/src/main/java/com/example/noram/MainActivity.java]: This should be email instead of phone number

File: code/noRAM/app/src/main/java/com/example/noram/MainActivity.java [hunk 3]
```diff
@@ -95,17 +98,24 @@ private void signInFirebase() {
...
                                     Boolean allowLocation = document.getBoolean("allowLocation");
-                                    attendee = new Attendee(user.getUid(), firstname, lastname, homepage, email, profilePicture, allowLocation);
+                                    Boolean defaultPhoto = document.getBoolean("defaultProfilePhoto");
+                                    attendee = new Attendee(user.getUid(), firstname, lastname, homepage, phoneNumber, allowLocation, defaultPhoto);
```
Comment (ethanchung936) [inline code/noRAM/app/src/main/java/com/example/noram/MainActivity.java]: This should also be email

**Review by ethanchung936 — APPROVED** [review]
Other than those two changes, looks good
